pom.xml

```diff
@@ -522,7 +522,7 @@
         <org.wso2.carbon.database.utils.version.range>[2.1.0,3.0.0)</org.wso2.carbon.database.utils.version.range>
 
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>7.1.0</carbon.identity.framework.version>
+        <carbon.identity.framework.version>7.7.2</carbon.identity.framework.version>
```

Comment:
Bump framework version once https://github.com/wso2/carbon-identity-framework/pull/6181 is merged.

Reply:
PR is closed since this is no longer needed.